code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -179,3 +179,29 @@
 	icon = 'icons/obj/eldritch.dmi'
 	icon_state = "eldrich_flask"
 	list_reagents = list(/datum/reagent/eldritch = 50)
+
+/obj/item/clothing/neck/crucifix
+	name = "crucifix"
+	desc = "In the eventuality that one of those you falesly accused is, in fact, a real witch, this will ward you against their curses."
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "crucifix"
+	icon_state = "eye_medalion"
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/clothing/neck/crucifix/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(slot == SLOT_NECK && istype(user))
+		ADD_TRAIT(user, TRAIT_WARDED, CLOTHING_TRAIT)
+		user.update_sight()
+
+/obj/item/clothing/neck/crucifix/dropped(mob/user)
```

Comment:
Is this called on dequip?

Reply:
`dropped` is called at `doUnEquip` and `put_in_hands`. It even is called on `Destroy` via `temporarilyRemoveItemFromInventory`.